### DISCUSSION TITLE
what ubuntu version

### DISCUSSION THREAD

**Discussion by hgarcia101685** [OpenSatKit/OpenSatKit#74] [Q&A] [answered]
I'm having issues getting Ruby gems installed on ubuntu 20.04.  
What version of Ubuntu have others used without issues?
Ruby version is 2.7.0, not sure if that makes a difference?

**Comment by dmccomas** [comment, marked as answer]
Did you get SSL errors?   

I just tried an install and got SSL errors.  I followed the instructions in the following stackoverflow post, rebooted, and reinstalled and it worked.  I tried to just do a bundle install after the ssl change I I ran into a JSON gem issue that I couldn't resolve whis is why I started over.  

https://stackoverflow.com/questions/62592042/unsupported-protocol-opensslsslsslerror-error-when-sending-mail-with-actio

**Reply by hgarcia101685** [reply]
I've just resolved my issues, thanks Dave!
For me it wasn't SSL, it was actually that I didn't follow instructions and didn't close out the terminal window before doing the bundle install.  
Once I closed terminal and opened a new one, the bundle install worked correctly.  
Thanks for responding!
